roomserver/storage/sqlite3/event_state_keys_table.go

```diff
@@ -136,7 +136,7 @@ func (s *eventStateKeyStatements) bulkSelectEventStateKey(
 	for k, v := range eventStateKeyNIDs {
 		iEventStateKeyNIDs[k] = v
 	}
-	selectOrig := strings.Replace(bulkSelectEventStateKeyNIDSQL, "($1)", common.QueryVariadic(len(eventStateKeyNIDs)), 1)
+	selectOrig := strings.Replace(bulkSelectEventStateKeySQL, "($1)", common.QueryVariadic(len(eventStateKeyNIDs)), 1)
```

Comment:
Please think of your future self and rename the constant! `bulkSelectEventStateKeyNIDSQL` should be `bulkSelectEventStateKeySQL` and vice versa.

Reply:
Aren't they right now? `bulkSelectEventStateKeySQL` for `bulkSelectEventStateKey` (as it returns a state key), `bulkSelectEventStateKeyNIDSQL` in `bulkSelectEventStateKeyNID` (as it returns a NID)?